test/e2e/pods.go

```diff
@@ -693,7 +693,8 @@ var _ = Describe("Pods", func() {
 									Port: intstr.FromInt(8080),
 								},
 							},
-							InitialDelaySeconds: 30,
+							InitialDelaySeconds: 15,
+							TimeoutSeconds:      30,
```

Comment:
Updated to `10s`

On Mon, Feb 8, 2016 at 2:58 PM, Vishnu Kannan <vishnuk@google.com> wrote:

> I'm fine with lowering it to 10s.
>
> On Mon, Feb 8, 2016 at 2:50 PM, Yu-Ju Hong <notifications@github.com>
> wrote:
>
>> In test/e2e/pods.go
>> <https://github.com/kubernetes/kubernetes/pull/20732#discussion_r52240259>
>> :
>>
>> > @@ -693,7 +693,8 @@ var _ = Describe("Pods", func() {
>> >  									Port: intstr.FromInt(8080),
>> >  								},
>> >  							},
>> > -							InitialDelaySeconds: 30,
>> > +							InitialDelaySeconds: 15,
>> > +							TimeoutSeconds:      30,
>>
>> I'm still unsure about bumping this from 1s to 30s. I think 10s is a
>> reasonable start, and if it still fails, the node may jsut be overloaded.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/kubernetes/kubernetes/pull/20732/files#r52240259>.
>>
>
>